test/blackbox/blackbox-kotlin.spec.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Blackbox tests are the final line of defence, that takes different real-life example documents and generate their corresponding models in all supported languages.
+ *
+ * For those languages where it is possible, the models are compiled/transpiled to ensure there are no syntax errors in generated models.
+ *
+ */
+
+import * as path from 'path';
+import * as fs from 'fs';
+import {
+  InputMetaModel,
+  InputProcessor,
+  KotlinFileGenerator,
+  KOTLIN_DEFAULT_PRESET, JAVA_COMMON_PRESET, KOTLIN_CONSTRAINTS_PRESET,
+} from '../../src';
+import { execCommand } from './utils/Utils';
+import filesToTest from './BlackBoxTestFiles';
+
+async function generate(fileToGenerateFor: string): Promise<InputMetaModel> {
+  const inputFileContent = await fs.promises.readFile(fileToGenerateFor);
+  const processor = new InputProcessor();
+  const input = JSON.parse(String(inputFileContent));
+  return processor.process(input);
+}
+
+function deleteDirectoryIfExists(directory: string) {
+  if (fs.existsSync(directory)) {
+    fs.rmSync(directory, { recursive: true });
+  }
+}
+
+describe.each(filesToTest)('Should be able to generate with inputs', ({ file, outputDirectory }) => {
+  const isWindows = process.platform === 'win32';
+  if (isWindows) {
+    // Windows environment has a weird setup, where it is using Kotlin Native instead of Kotlin JVM as it's compiler
+    // (See https://github.com/asyncapi/modelina/issues/1080)
+    return;
+  }
```

Comment:
Was a bit too greedy without checking if it worked 😅 Adapted it for Jest to just skip the tests conditionally instead, solving the black-box error.

```suggestion
const isWindows = process.platform === 'win32';
const describeIf = (condition: boolean) => condition ? describe : describe.skip;

// Windows environment has a weird setup, where it is using Kotlin Native instead of Kotlin JVM as it's compiler
// (See https://github.com/asyncapi/modelina/issues/1080)
describeIf(!isWindows).each(filesToTest)('Should be able to generate with inputs', ({ file, outputDirectory }) => {
```